### DISCUSSION TITLE
Valley-channel width ratio application to binary IP model outputs

### DISCUSSION THREAD

**Discussion by smnorris** [smnorris/bcfishpass#251] [Q&A]
I am not sure if/how we can apply this measure to the existing binary spawning/rearing models. For highly confined Coho streams [Burnett 2007](https://www.fs.fed.us/pnw/lwm/aem/docs/burnett/2007_burnett_reeves_ecoapp_intrinspoten.pdf) only bumps the IP model down to .2, it does not exclude the streams entirely.

- are there ratio thresholds at which we can presume spawning/rearing do not occur (or are minimal enough to exclude)?
- if not, do we need to modify the data model to support relative percentage scores for spawning/rearing potential?

I'll have to generate some valley-width index numbers and compare to known rearing/spawning locations as a basis for discussion.

**Comment by nickw-CWF** [comment]
The other parameters (gradient and MAD) are also continuous variables in traditional IP models. For setting existing thresholds, I compared scientific literature values for spawning/rearing with the high/medium value ranges from existing IP models, while excluding "low" ranges for each variable, to turn it into a binary measure of habitat potential. Part of this was to exclude low value habitat from consideration, but it was also to support quantification of the habitat connectivity status for a watershed. It is more involved to calculate a % connected if each stream segment has a suitability rating between 0-1, rather than true/false.

We could try and set min./max. thresholds on confinement values to maintain the model structure. See [Sheer et al. 2009](https://www.researchgate.net/publication/261359826_Development_and_Management_of_Fish_Intrinsic_Potential_Data_and_Methodologies_State_of_the_IP_2008_Summary_Report_Pacific_Northwest_Aquatic_Monitoring_Partnership_National_Oceanic_and_Atmospheric_Admi) for how channel confinement is used in other IP models.

Another option would be to also convert gradient and discharge to suitability scores (0-1), so that each stream segment gets assigned an overall habitat suitability score and then we set a cut off on for what we want to consider "suitable" habitat for quantification purposes (i.e., an overall score > 0.65 = potential suitable habitat).

I can talk to Nick L. about this, but it might be a good topic to bring to the group next week. Let me know if you'd like to hop on a quick call to discuss further.

**Reply by smnorris** [reply]
Thanks. I would lean towards option 1 for the short term but wondered if it is possible to define a valley-width index that rules out habitat potential. We will have to look at the data.  That is a great reference for summarizing approaches.
